system-test/logging-bunyan.ts

```diff
@@ -16,31 +16,38 @@
 
 import * as assert from 'assert';
 import * as bunyan from 'bunyan';
-import delay from 'delay';
-
+import * as uuid from 'uuid';
 import * as types from '../src/types/core';
 import {ErrorsApiTransport} from './errors-transport';
 
 const {Logging} = require('@google-cloud/logging');
 const logging = new Logging();
 import {LoggingBunyan} from '../src/index';
+import delay from 'delay';
 
-const LOG_NAME = 'bunyan_log_system_tests';
+const WRITE_CONSISTENCY_DELAY_MS = 90000;
 
-describe('LoggingBunyan', () => {
-  const WRITE_CONSISTENCY_DELAY_MS = 90000;
+const UUID = uuid.v4();
```

Comment:
Being picky, but couldn't this all be boiled down to `const LOG_NAME = \`${uuid.v4()}-logging-bunyan-system-test\`;`?  It looks like we have a method that only gets called once with a single string.

Reply:
I think the expectation is that we can have have more tests in this file in the future. They could potentially want to use unique log names.

The pedigree of this code is that is copied from sisten winston library, which indeed has multiple tests.

I can make the change if you insist, but I do not think it is necessary.

Reply:
I do not insist :)